modules/latentsvm/test/test_latentsvmdetector_cascade.cpp

```diff
@@ -127,13 +127,14 @@ bool compareResults( const std::vector<lsvm::LSVMDetector::ObjectDetection>& cal
 
 void LatentSVMDetectorCaskadeTest::run( int /* start_from */)
 {
-    std::string img_path_cat = std::string(ts->get_data_path()) + "cat.png";
-    std::string img_path_cars = std::string(ts->get_data_path()) + "cars.png";
+	std::string test_data_path = ts->get_data_path() + "latentsvmdetector/";
```

Comment:
Yes it is a typo, but the complete LatentSVM module has this problem of language mistakes. You could change it but then do a replace all of all occurences in files accross the repository in order to ensure that you do not break other stuff.